lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
```

Comment:
@mikemcquaid You may be against this, as I need to import Homebrew's `version` to do the version comparison.